.github/workflows/docker.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Docker
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
```

Comment:
Similarly here, I don't think we need a docker image build for every push -- perhaps this can be limited to tags, as well -- or perhaps to releases?

Reply:
What this achieves is that you have a `main` Docker tag that always points to the most recent version of the repository. It is interesting in case someone wants to test what is in the repository right now (which is not yet associated with any release or tag). This docker tag is overwritten each time, so it does not take up more space.